.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,81 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="PyPackageRequirementsInspection" enabled="true" level="WARNING" enabled_by_default="true">
+      <option name="ignoredPackages">
+        <value>
+          <list size="47">
+            <item index="0" class="java.lang.String" itemvalue="Aspose.Slides" />
```

Comment:
The value 'Aspose.Slides' is not properly formatted. It should be 'aspose.slides'.